traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -27,104 +29,103 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/lib/pq"
-
 	"github.com/apache/trafficcontrol/lib/go-log"
 	"github.com/apache/trafficcontrol/lib/go-tc"
 	"github.com/apache/trafficcontrol/lib/go-tc/tovalidate"
 	"github.com/apache/trafficcontrol/lib/go-util"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/auth"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/dbhelpers"
-
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/deliveryservice"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/tenant"
-	"github.com/go-ozzo/ozzo-validation"
+
+	validation "github.com/go-ozzo/ozzo-validation"
```

Comment:
Is this necessary? The package name is already `validation`.

Reply:
This was done automatically by `gopls`. I believe the tool does that because the package name differs significantly from the the last part of the import path. I guess the tool is trying to show the actual package name explicitly in the code since the package doesn't follow the Go convention of the package name matching the last segment of the import path.

Reply:
I don't think a language server ought to be making stylistic changes without user input or configuration in the tools that use it, but I suppose this isn't hurting anything. Just to be clear though, we don't enforce `gopls` suggestions/edits.